CHANGES/24.feature.rst

```diff
@@ -0,0 +1 @@
+Restored the ability to import :func:`propcache.api.under_cached_property` and :func:`propcache.api.cached_property` directly from ``propcache`` -- by :user:`bdraco`.
```

Comment:
If this is replacing the previous release attempt that never happened, this change note will no longer refer to something that existed in the previous version. Hence, I'd rather modify the breaking change in the log, perhaps.

Reply:
I'll revert the release since it never happened anyways, than rebase this one and we can adjust the messaging a bit

https://github.com/aio-libs/propcache/pull/28

Reply:
I reworked the message a bit.

Reply:
<img width="674" alt="Screenshot 2024-10-06 at 2 02 14 PM" src="https://github.com/user-attachments/assets/0e461f39-6a08-462b-b673-14325a771242">

Reply:
nice!